src/app/Instance/BridgeOverview/Components/OBEmptyState.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  Button,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  Text,
+  TextContent,
+  Title,
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+import React from "react";
```

Comment:
It's a good habit to place react imports on top of the file. Just to be able to locate things easily.
We never formalized some guidelines about how to sort imports. We probably should!